### PR TITLE
Fixes Bug #1281520

### PR DESCRIPTION
**Pull request by vedantrathore**
Removed whitelisting suppport

### CODE REVIEW

**Comment by vedantrathore** [comment]
@bhearsum could you review this?

**Comment by vedantrathore** [comment]
@bhearsum I couldn't figure out why the build is failing, all the tests pass in the logs

File: auslib/test/web/test_client.py [hunk 1]
```diff
@@ -669,35 +606,6 @@ def testVersion4Get(self):
...
-        ret = self.client.get('/update/5/b2g/1.0/1/p/l/foxfood/a/a/a/000000000000009/update.xml')
-        self.assertUpdateEqual(ret, """<?xml version="1.0"?>
-<updates>
-    <update type="minor" version="None" extensionVersion="2.5" buildID="25">
-        <patch type="complete" URL="http://a.com/public" hashFunction="sha512" hashValue="23" size="22"/>
-    </update>
-</updates>
-""")
 
     def testVersion6GetWithSystemCapabilitiesMatch(self):
```
Comment (NinadBhat) [inline auslib/test/web/test_client.py]: @vedantrathore your build failed as you have two blank lines above this function(there should only be one).
You can have a look at the log statement [here.](https://taskcluster.github.io/unified-logviewer/?url=https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FFd_LNBN0TuqS4auGoquqKg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log&highlightStart=3277&highlightEnd=3277&lineNumber=3278&wrapLines=false&showLineNumbers=true&jumpToHighlight=true&followLog=false&lineHeight=19&asText=false)

Comment (vedantrathore) [inline auslib/test/web/test_client.py]: Oh okay, thanks! @NinadBhat

**Comment by vedantrathore** [comment]
@bhearsum I think it looks good to merge. :)

**Comment by mozbhearsum** [comment]
@vedantrathore - I'm closing this due to inactivity. Feel free to re-open if you want to continue working on it.

**Pull request closed without merging**